common_spree_dependencies.rb

```diff
@@ -3,6 +3,7 @@
 # the one component of Spree.
 source 'https://rubygems.org'
 
+gem 'rails', '4.2.0'
```

Comment:
Yup 3.0.x was our rails 4.2.x upgrade.  I stole your patch so that shouldn't be an issue:
https://github.com/spree/spree/blob/3-0-stable/core/spec/support/rspec-activemodel-mocks-patch.rb